Fabric.Identity.API/Persistence/SqlServer/Scripts/create_identity_tables.sql

```diff
@@ -1,3 +1,5 @@
+USE [Identity]
```

Comment:
Do we still need this SQL script?

Reply:
@kpaul - can you clarify if this script is needed?

Reply:
i dont think we need it anymore as all the db entities are created via the dacpac